File: Leetcode/1823-find-the-winner-of-the-circular-game/1823-find-the-winner-of-the-circular-game.cpp
class Solution {
public:
    int findTheWinner(int n, int k) {
        vector<int>arr;
        for(int i=1;i<n+1;i++)
            arr.push_back(i);
        int tmp=0;
        while(arr.size()>1){
            tmp=(tmp+k-1)%arr.size();
            arr.erase(arr.begin()+tmp);
            
        }
        return arr[0];
    }
};

File: Leetcode/1823-find-the-winner-of-the-circular-game/1823-find-the-winner-of-the-circular-game.py
from queue import Queue
class Solution:
    def findTheWinner(self, n: int, k: int) -> int:
        l=[i+1 for i in range(n)]
        tmp=0
        while len(l)>1:
            tmp=(tmp+k-1)%len(l)
            l.pop(tmp)
        return l[0]
            
                
                
            
            